symplyphysics/laws/electricity/circuits/resonators/resonant_frequency_of_rectangular_resonator.py

```diff
@@ -26,37 +27,40 @@
     clone_as_symbol,
 )
 
-resonant_frequency = symbols.temporal_frequency
+resonant_frequency = clone_as_symbol(symbols.temporal_frequency, display_symbol="f_r", display_latex="f_\\text{r}")
 """
 Resonant :symbols:`temporal_frequency` of the resonator.
 """
 
 first_index = clone_as_symbol(symbols.positive_number, subscript="1")
 """
-First index, see :symbols:`positive_number`.
+First index that changes along the :attr:`~length` of the resonator, see
+:symbols:`positive_number`.
 """
 
 second_index = clone_as_symbol(symbols.positive_number, subscript="2")
 """
-Second index, see :symbols:`positive_number`.
+First index that changes along the :attr:`~width` of the resonator, see
+:symbols:`positive_number`.
 """
 
 third_index = clone_as_symbol(symbols.positive_number, subscript="3")
 """
-Third index, see :symbols:`positive_number`.
+First index that changes along the :attr:`~height` of the resonator, see
```

Comment:
First index everywhere. Also they should be in order m, n, p, i.e. width, height, length. Also mention m, n or p index in each of the index description. Might be a good idea to put m, n, p in the symbol subscript.